corehq/apps/userreports/tasks.py

```diff
@@ -54,7 +54,7 @@ def _build_indicators(indicator_config_id, relevant_ids):
         redis_client.delete(redis_key)
 
 
-@task(queue='ucr_queue', ignore_result=True, acks_late=True)
+@task(queue='ucr_queue', ignore_result=True)
 def rebuild_indicators(indicator_config_id):
```

Comment:
My understanding is that this task drops any existing table for that data source, that starts populating from scratch.
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/sql/adapter.py#L86
If they clicked the button before the task had finished, but the task had been killed, it would start over. I think if the task was actively running, sqlalchemy would throw a connection error on the second rebuild task so there would be no negative consequences.